src/main/java/com/ericsson/ei/mongo/MongoDBHandler.java

```diff
@@ -364,33 +365,27 @@ private boolean doDrop(String dataBaseName, String collectionName, MongoQuery qu
 
     }
 
-    private MongoCollection<Document> getMongoCollection(String databaseName,
-            String collectionName) {
+    private MongoCollection<Document> getMongoCollection(String databaseName, String collectionName)
+            throws MongoClientException {
         if (mongoClient == null) {
-            return null;
+            throw new MongoClientException("Failed to connect MongoDB");
```

Comment:
MongoDB is used from many places, all test must pass and manual tests must be made before merging those changes.

Reply:
ok